xml/security_ldap.xml

```diff
@@ -721,39 +639,10 @@ MAY (userPassword $ searchGuide $ seeAlso $ businessCategory <co xml:id="co.ldap
  <sect1 xml:id="sec.security.ldap.slapd" os="sles;osuse">
   <title>Manually Configuring an LDAP Server</title>
 
-<!--
-  <para>
-   Your installed system contains a complete configuration file for your
-   LDAP server at <filename>/etc/openldap/slapd.conf</filename>. The single
-   entries are briefly described here and necessary adjustments are
-   explained. Entries prefixed with a hash (#) are inactive. This comment
-   character must be removed to activate them.
-  </para>
--->
-
-  <para>
-   &yast; uses OpenLDAP's dynamic configuration database
-   (<systemitem>back-config</systemitem>) to store the LDAP server's
-   configuration. For details about the dynamic configuration back-end, see
-   the <systemitem>slapd-config(5)</systemitem> man page or the OpenLDAP
-   Software 2.4 Administrator's Guide located at
-   <filename>/usr/share/doc/packages/openldap2/guide/admin/guide.html</filename>
-   on your system if the <systemitem>openldap2</systemitem> package is
-   installed.
-  </para>
-
-  <tip>
-   <title>Upgrading an Old OpenLDAP Installation</title>
-   <para>
-    &yast; does not use <filename>/etc/openldap/slapd.conf</filename> to
-    store the OpenLDAP configuration anymore. In case of a system upgrade, a
-    copy of the original <filename>/etc/openldap/slapd.conf</filename> file
-    will get created as
-    <filename>/etc/openldap/slapd.conf.YaSTsave</filename>.
-   </para>
-  </tip>
-
   <para>
+   <remark>taroth 2019-02-21: @wbrown: the content of this section needs to be
+   revised for sure - could you please let me know what to remove, replace, or
```

Comment:
Basically everything. 389-ds works really differently to openldap, so we probably need a completely different approach. Perhaps a good place to start is from my quickstart docs I have made? http://www.port389.org/docs/389ds/howto/quickstart.html

I also think perhaps it may be good to have a list of items we want to have documented. Probably at least:

* Setup/Install
* Basic admin (users, groups, setting up a client with sssd)
* Backup/Restore
* Setting up a replica
* Decommisioning a server?

Is there more? What points are covered here?

Reply:
Thanks, William! Sounds good to me. Also your quick start is a good basis to start from. :)

 Regarding the LDAP chapter, I could imagine the following structure:
- Structure of an LDAP Directory Tree (keep)
- Configuring the 389-ds LDAP Server (new)
    * Installing the Software
    * Setting Up the Server
    * Administering the Server (users, groups, setting up a client with sssd)
    * Backing Up and Restoring the Database
- Configuring an LDAP Client (new?/ adjust former section?)

Depending on which of these steps can also be done with YaST, we need to describe both approaches for those steps (manually & via YaST). 

Not sure about the topics 'replica' and 'decommissioning' yet, let's discuss this next week. 
I would try to focus on the basic tasks for the chapter. For further configuration tasks, we can probably also refer to the upstream docs.

Reply:
For "structure", I'm happy for you to use content from my personal blog about this topic https://fy.blackhats.net.au/blog/html/pages/ldap_guide_part_1_foundations.html  (I often hear this is the "best" ldap primer on the internet :) )

Configuration sounds good, and seems pretty similar to the quickstart. We can go into more detail if we want.

I think the "ldap client" setup, is in the quickstart I linked too, and certainly we should expand on that too.

I'm not sure about yast setup (I haven't been involved in that part), but certainly the manual setup should be pretty easy.

I think we do this first, then we'll add replication later? Replication is our main selling point as an LDAP implementation, so it would be great to have something in the suse docs about this.

Reply:
@Firstyear : Actually, regarding the section 'Structure of an LDAP Directory Tree' my thought was to keep the section that we already have in the current chapter. If that is not good enough from your point of view, we can probably add information from your LDAP guide.

About replication: If this is the main selling point, we should certainly cover it in the docs (how soon is probably a question of resources).

Reply:
I think there are some improvements to make it it's clarity, it's super dense at the moment. But perhaps that's a future change? I just worry if this is someones first attempt to understand LDAP, would they read this and understand how it works?  The current content is okay for now (it took me a while to convince github to let me read it).

Reply:
I'll keep this section on the list of ToDos then, but it has rather low priority, because first of all, we need to update the other content of the chapter that no longer applies. 

Regarding readability: As reading longer texts on GH is cumbersome, I will provide a draft PDF of the updated chapter as soon as we have reached a version that makes sense for you to read. :) Until then, you can also have a look at the  HTML output of the content in the develop branch, which was the starting point of this feature branch (see the links to docserv I sent you).

Reply:
Sounds good, I'll look at those links soon. I agree that we can leave this as low priority :)